TESTING.md

```diff
@@ -326,19 +326,27 @@ Terminate**.
 
 **6.43** ASSERTION: A Scala REPL for **O1Library** opens.
 
-**6.44** Close the REPL.
+**6.44** Execute the following command in the REPL: `println("Myöhässä")`
+
+**6.45** ASSERTION: The word appears correctly, without any garbled letters.
+
+**6.46** Execute the following command in the REPL: `println("世界です")`
```

Comment:
Combining these strings into a single input would make it a bit faster to go through.

Reply:
Good point